pkg/combustion/registry.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/suse-edge/edge-image-builder/pkg/fileio"
 	"github.com/suse-edge/edge-image-builder/pkg/image"
 	"github.com/suse-edge/edge-image-builder/pkg/log"
+	registry "github.com/suse-edge/edge-image-builder/pkg/manifests"
```

Comment:
Package name and directories must be the same. This is why you're required to use an alias here and you wouldn't be.